### ISSUE TITLE
mono support

### ISSUE THREAD

**Issue by naveensrinivasan** [RolandPheasant/TailBlazer#61]
It would be cool to have mono support which could make this work on non-windows platform.

**Comment by RolandPheasant** [comment]
Definitely have to look into this. Have you ever built code for Mono before? I ask as I would need help

**Comment by naveensrinivasan** [comment]
@RolandPheasant  Yes I have. Here is a recent one for octokit https://github.com/octokit/octokit.net/pull/995.

**Comment by RolandPheasant** [comment]
Cheers. I will take a look over xmas. And probably come back with a load of questions

**Comment by jkoritzinsky** [comment]
Since WPF isn't available on Mono, you could look at [Perspex](GitHub.com/Perspex/Perspex). It's a cross-platform WPF-style UI toolkit that's currently in alpha. That way there would be minimal needed change in the codebase since WPF idioms still work in many cases.
